github/spec/features/configure_repos_spec.rb

```diff
@@ -41,8 +51,25 @@ def given_theres_a_repo
   end
 
   def and_the_repo_has_a_jenkinsfile
+    payload = {
+      name: "Jenkinsfile",
+      content: "",
+      encoding: "base64",
+    }
+
+    stub_request(:get, "https://api.github.com/repos/alphagov/publishing-api/contents/Jenkinsfile").
+      to_return(body: payload.to_json, headers: { content_type: "application/json" }, status: 200)
+  end
+
+  def and_the_repo_has_a_jenkinsfile_with_e2e_tests
+    payload = {
+      name: "Jenkinsfile",
+      content: "IyEvdXNyL2Jpbi9lbnYgZ3Jvb3Z5Cgpub2RlIHsKICBkZWYgZ292dWsgPSBs\nb2FkICcvdmFyL2xpYi9qZW5raW5zL2dyb292eV9zY3JpcHRzL2dvdnVrX2pl\nbmtpbnNsaWIuZ3Jvb3Z5JwogIGdvdnVrLmJ1aWxkUHJvamVjdCgKICAgIGV4\ndHJhUGFyYW1ldGVyczogWwogICAgICBzdHJpbmdQYXJhbSgKICAgICAgICBu\nYW1lOiAiUFVCTElTSElOR19BUElfUEFDVF9CUkFOQ0giLAogICAgICAgIGRl\nZmF1bHRWYWx1ZTogImRlcGxveWVkLXRvLXByb2R1Y3Rpb24iLAogICAgICAg\nIGRlc2NyaXB0aW9uOiAiVGhlIGJyYW5jaCBvZiBQdWJsaXNoaW5nIEFQSSBw\nYWN0IHRlc3RzIHRvIHJ1biBhZ2FpbnN0IgogICAgICApLAogICAgXSwKICAg\nIGJlZm9yZVRlc3Q6IHsKICAgICAgZ292dWsuc2V0RW52YXIoIlBBQ1RfQlJP\nS0VSX0JBU0VfVVJMIiwgImh0dHBzOi8vcGFjdC1icm9rZXIuY2xvdWRhcHBz\nLmRpZ2l0YWwiKQogICAgfSwKICAgIHB1Ymxpc2hpbmdFMkVUZXN0czogdHJ1\nZSwKICAgIGFmdGVyVGVzdDogewogICAgICBnb3Z1ay5ydW5SYWtlVGFzaygi\ncGFjdDp2ZXJpZnk6YnJhbmNoWyR7ZW52LlBVQkxJU0hJTkdfQVBJX1BBQ1Rf\nQlJBTkNIfV0iKQogICAgfQogICkKfQo=\n",
```

Comment:
Would encoding the string in the test make it more readable?

```
content: Base64.encode64(...)
```

Reply:
Yeah, potentially. It would make that line even longer, but maybe that's fine.